plugin/kubernetes/handler_test.go

```diff
@@ -206,10 +207,10 @@ func (APIConnServeTest) PodIndex(string) []interface{} {
 	return a
 }
 
-func (APIConnServeTest) ServiceList() []*api.Service {
+func (APIConnServeTest) ServiceList() []interface{} {
```

Comment:
Why does this need to be moved to empty interface slice? We loose all type safety